vminfo_parser/config.py

```diff
@@ -202,3 +207,28 @@ def _validate(self: t.Self) -> None:
         if not hasattr(self, "file"):
             LOGGER.critical("File not specified in yaml or command line")
             exit(1)
+
+    @classmethod
```

Comment:
Not sure how I missed this earlier, but this shouldn't be a classmethod.  It is should reference self instead of an instance of the class passed to it.

Reply:
This has been addressed. I think I meant to have it be a static method because it really doesn't use anything from the class. But it doesn't hurt it to be a regular class method. Either way you were correct, classmethod was a mistake